product-service/model/create_samples.sql

```diff
@@ -0,0 +1,42 @@
+create extension if not exists "uuid-ossp";
+
+drop table if exists products cascade;
+
+create table products (
+  id uuid primary key default uuid_generate_v4(),
+  title text not null,
+  description text,
+  price integer
+);
+
+drop table if exists stocks;
+
+create table stocks (
+  id serial primary key,
```

Comment:
should not be additional id according to requirements

Reply:
@agarmashevska I don't think that Product or Stock tables should strictly adhere to the schemas, described in the task. As the course creators said: 'Your model/site can be whatever you want'. So it's nothing bad that he added 1 additional field to the model.

Reply:
@agarmashevska, @Emerkof
I would say more. It is good practice and id should be present for every present in real life. But for this task we use such a simplification and it is ok not to add id for now. https://www.youtube.com/watch?v=7zd3K6zqmmE&feature=youtu.be&t=3111 - here they explain it.